src/tests/grid/test_cubedsphere.cc

```diff
@@ -0,0 +1,71 @@
+#include "atlas/functionspace/NodeColumns.h"
+#include "atlas/grid.h"
```

Comment:
Remove Partitioner.h and add the text header with licence.

Reply:
What about including 
```
#include "atlas/array/MakeView.h"
#include "atlas/field/Field.h"
```
You are using them, but they are hidden by auto

Reply:
Done. Although it's reasonable to assume that `NodeColumns.h` will include the definition of a field, given it returns a non-pointer field.